encryption/starter/main.go

```diff
@@ -14,21 +14,20 @@ import (
 func main() {
 	// The client is a heavyweight object that should be created once per process.
 	c, err := client.Dial(client.Options{
-		// If you intend to use the same encryption key for all workflows you can
-		// set the KeyID for the encryption encoder like so:
-		//
-		// Set DataConverter to ensure that workflow inputs and results are
-		// encrypted/decrypted as required.
+		// If you intend to let the dataConverter to decide encryption key for all workflows
+		//you can set the KeyID for the encryption encoder like so:
 		//
 		//   DataConverter: encryption.NewEncryptionDataConverter(
 		// 	  converter.GetDefaultDataConverter(),
 		// 	  encryption.DataConverterOptions{KeyID: "test", Compress: true},
 		//   ),
 		//
 		// In this case you do not need to use a ContextPropagator.
+		// You also can implement the EncryptionDataConverter to decide the encryption key
```

Comment:
```suggestion
		// You also can implement the data converter to decide the encryption key
```
What is `EncryptionDataConverter` referring to?

Reply:
In `data_converter.go` the constructor name is `NewEncryptionDataConverter`. I thought that's what you want to call it to be different from regular dataConverter.

Reply:
No need for proper name

Reply:
that's fine, I changed to dataConverter